rpm/create-rpm.sh

```diff
@@ -21,7 +21,7 @@
 
 CEP_VERSION=$2
 if [ -z "$CEP_VERSION" ]; then
-  CEP_VERSION=1.1.0-SNAPSHOT
+  CEP_VERSION=1.2.0
```

Comment:
Not sure if we still using this script... but in case of doubt I'm also changing it.

NTC